### PR TITLE
sample(spanner): Update samples for dml returning

### PR DESCRIPTION
**Pull request by diptanshumittal**

### CODE REVIEW

File: spanner/spanner_samples.rb [hunk 1]
```diff
@@ -200,13 +200,16 @@ def create_database_with_encryption_key project_id:, instance_id:, database_id:,
                                      SingerId     INT64 NOT NULL,
                                      FirstName    STRING(1024),
                                      LastName     STRING(1024),
-                                     SingerInfo   BYTES(MAX)
+                                     SingerInfo   BYTES(MAX),
+                                     FullName STRING(2048) AS
+                                     (ARRAY_TO_STRING([FirstName, LastName]
```
Comment (rajatbhatta) [inline spanner/spanner_samples.rb:205]:
```suggestion
                                     (ARRAY_TO_STRING([FirstName, LastName], \" \")) STORED
```

**Comment by rajatbhatta** [comment]
Hi @diptanshumittal: Thank you for working on this. Just one comment from my side: Can we copy the code comments from samples here: https://github.com/googleapis/java-spanner/pull/2007/files. This will keep code comments consistent as well.

**Review by rajatbhatta — APPROVED** [review]
LGTM.

**Review by SandeepTuniki — APPROVED** [review]
LGTM

**Pull request merged**